### PR TITLE
New crypto-conditions

### PR DESCRIPTION
**Pull request by justmoon**
This prepares Five Bells Shared for migrating Five Bells to the new crypto-conditions format. See https://github.com/interledger/five-bells-condition

### CODE REVIEW

**Comment by emschwartz** [comment]
LGTM

File: lib/config.js [hunk 1]
```diff
@@ -181,8 +182,9 @@ function parseED25519 (prefix) {
       throw new ServerError('No ' + (prefix ? prefix.toUpperCase() + '_' : '') +
         'ED25519_SECRET_KEY provided.')
     }
-    keyPair = tweetnacl.sign.keyPair()
-    ed25519.secret = tweetnacl.util.encodeBase64(keyPair.secretKey)
+    const seed = crypto.randomBytes(32)
+    keyPair = tweetnacl.sign.keyPair.fromSeed(seed)
+    ed25519.secret = seed.toString('base64')
```
Comment (clark800) [inline lib/config.js]: In profiling we noticed that the tweetnacl base64 encoding seemed slow, so I'm glad you used the node built-in. Should we do the same for the public key below?

Comment (justmoon) [inline lib/config.js]: Fixed.

**Comment by clark800** [comment]
LGTM

**Pull request merged**